src/stringClass.js

```diff
@@ -32,7 +32,7 @@ const stringClass = {
 
 /**
    * This method takes a string and returns its first letter in uppercase
-   * @param {String} this
+   * It extends the String.prototype to achieve this
```

Comment:
This is looking redundant.

Can we leave it out an have a meaningful comment when we're actually assigning the methods?